README.md

```diff
@@ -283,3 +283,130 @@ databases, but to continue a single mainline/unstable database.
   new mainline development version (perhaps using the `-alpha` suffix; in our
   example, `0.3.6-alpha`).  See
   [#1155](https://github.com/PolymerLabs/arcs/pull/1155) for an example.
+
+## Hosted
+
+
```

Comment:
Add a TODO noting to write and link to a design/architecture doc so that curious folk can get high level info on what the structure looks like.